### ISSUE TITLE
Cannot install M2Crypto 0.37

### ISSUE THREAD

**Issue by petricm** [DIRACGrid/DIRACOS#158]
With the version of M2Crypto 0.37.0 swig 2.0.10 is required this version exists only on CC7.

**Comment by petricm** [comment]
The swig problem can temporarily be circumvented by 
```
yum --enablerepo=PUIAS_6_computational install -y swig2
ln -s /usr/bin/swig2 /usr/bin/swig
```

**Comment by petricm** [comment]
will not fix, M2Crypto stays on 0.36.0 until end of life of DIRACOS
